### DISCUSSION TITLE
Screenshot: Filter out certain untradeable drops

### DISCUSSION THREAD

**Discussion by NotBlackice** [runelite/runelite#15632] [Ideas]
Just noticed I had nearly 1.3gbs in screenshots of hallowed marks because you get them every time you loot in sepulchre. Also had a ton of screenshots of the book drops from ToA. I'm sure there are others that would be nice to exclude.

**Comment by geheur** [comment]
ToB would also be good to exclude. It drops books from every boss too, I bet those cause screenshots too.

**Comment by RsFoe** [comment]
having more customizability would be super nice for the screenshot plugin, whether its a list to exclude bosses/chests or something. I like the plugin but hate that it screenshots bosses bosses like dks and mole when I want to only turn it on for other bosses so the folder includes a ton of useless screenshots
